src/cargo/core/features.rs

```diff
@@ -216,7 +216,7 @@ impl Edition {
         match self {
             Edition2015 => false,
             Edition2018 => true,
-            Edition2021 => false,
+            Edition2021 => true,
```

Comment:
This comment above the function also needs updating: :)
```rust
/// Ideally this would not be necessary, but currently 2021 does not have
/// any lints, and thus `rustc` doesn't recognize it. Perhaps `rustc`
/// could create an empty group instead?
```

Reply:
Oh, thanks!  I actually had a change in my editor and somehow didn't hit save.